docs/management/cases/manage-cases.asciidoc

```diff
@@ -27,6 +27,85 @@ information, refer to <<add-case-connectors>>.
 
 . After you've completed all of the required fields, click *Create case*.
 
+[[case-custom-fields]]
+=== Add custom fields
+
+preview::[]
+
+You can add optional and required fields for customized case collaboration.
+
+. Go to *{stack-manage-app} > Cases* and click *Settings*.
++
+--
+[role="screenshot"]
+image::images/cases-custom-fields-view.png[View custom fields in case settings]
+// NOTE: This is an autogenerated screenshot. Do not edit it directly.
+
+NOTE: To view and change case settings, you must have the appropriate {kib} feature privileges. Refer to <<setup-cases>>.
+--
+
+. In the *Custom fields* section, click *Add field*.
++
+--
+[role="screenshot"]
+image::images/cases-custom-fields-add.png[Add a custom field in case settings]
+// NOTE: This is an autogenerated screenshot. Do not edit it directly.
+--
+
+. Enter a field label.
+
+. Choose a field type: text or toggle.
+
+. If you want the text field to be mandatory in all cases, select *Make this field required*.
+
+. Click *Save field*.
+
+You can subsequently remove or edit custom fields on the *Settings* page.
+
+WARNING: If you remove a custom field, it is removed from all cases.
+The field values that you added to your cases are no longer accessible.
```

Comment:
This is true for the UI but not for the API. If you had configured a custom field, set it on a case, and then you remove it the APIs will return the value of the deleted custom field for the particular case. In the future, we want to delete the data from the Case. I am not sure if we should mention it. It can be confusing to our users and I am not sure what value brings.

Reply:
> ... I am not sure if we should mention it. It can be confusing to our users and I am not sure what value brings.

If I understand correctly, you're recommending that we remove this warning. Since it seems like the warning is not true depending on whether you use the UI or API, I agree it's confusing so I'll remove it. If I've misunderstood and you want something else removed, just let me know.

Reply:
Removed warning in 190650610a0a120432345b3a14cd1d7e0e9c6b69